.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 21.0
-elixir 1.7.3-otp-21
+erlang 21.2.2
+elixir 1.8.0-rc.0
```

Comment:
not sure how you feel about this, but somehow that to use this feature you got to really asdf install 1.7.3-otp-21 just having 1.7.3 and otp 21 switched on at the same time kinda irks me the wrong way. Might just be me though woman_shrugging